mlctl/jobs/MlctlBatchJob.py

```diff
@@ -12,8 +12,8 @@ def __init__(self, job_type, project, name=None):
             self.name = name
         else:
             # else make a new name randomly
-            words = RandomWords().get_random_words()
-            self.name = f'mlctl-batch-{words[1]}'
+            word = RandomWords().random_word()
+            self.name = f'mlctl-batch-{word}'
```

Comment:
Batch jobs aren't working yet FYI. It's on the to do list.

Reply:
Yup, just making it consistent while at it